packages/@okta/vuepress-theme-prose/components/Hero.vue

```diff
@@ -1,25 +1,74 @@
 <template>
   <!-- Remove style attribute from .hero section when removing promo banner -->
-  <section
-    class="hero"
-    style="padding-top: 28px; padding-bottom: 28px;"
-  >
-    <div class="hero__wrapper wrapper">
-      <h1 class="hero__title dont-break-out">
-        Okta developer
-      </h1>
-      <p class="hero__paragraph dont-break-out">
-        Our developer portal enables you to deploy auth that protects your users, apps, APIs, and infrastructure.
-      </p>
-      <!-- Start workshops promo banner -->
-      <p style="color: white; font-size: 18px; line-height: 1.4; max-width: 500px;">
-        Get your app <a
-          style="color: white; text-decoration: underline;"
-          href="https://regionalevents.okta.com/enterprisereadyworkshops"
-          target="_blank"
-        >enterprise-ready with free virtual workshops</a>!
-      </p>
-      <!-- End workshops promo banner -->
-    </div>
-  </section>
+  <div class="hero-container">
+    <section
+      class="hero"
+      style="padding-top: 28px; padding-bottom: 28px;"
+    >
+      <div class="hero-wrapper-container">
+        <div :class="{ 'hero__wrapper': true, 'wrapper': true, 'full-width-wrapper': !showBanner }">
+          <h1 class="hero__title dont-break-out">
+            Okta developer
+          </h1>
+          <p class="hero__paragraph dont-break-out">
+            Our developer portal enables you to deploy auth that protects your users, apps, APIs, and infrastructure.
+          </p>
+          <!-- Start workshops promo banner -->
+          <p style="color: white; font-size: 18px; line-height: 1.4; max-width: 500px;">
+            Get your app <a
+              style="color: white; text-decoration: underline;"
+              href="https://regionalevents.okta.com/enterprisereadyworkshops"
+              target="_blank"
+            >enterprise-ready with free virtual workshops</a>!
+          </p>
+          <!-- End workshops promo banner -->
+        </div>
+        <!-- 
+          For understanding usage of the banner, refer https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/2920874189/Adding+a+banner+on+homepage+in+dev+docs
+         -->
+        <HomePageBanner
+          v-if="showBanner"
+          primary-cta-text="Primary button"
+          primary-cta-link="/docs/guides/"
+          primary-link-target="_self"
+          secondary-cta-text="Secondary button"
+          secondary-cta-link="https://www.yahoo.com"
+          @onPrimaryCtaClick="handlePrimaryCtaClick()"
+          @onSecondaryCtaClick="handleSecondaryCtaClick()"
```

Comment:
onClick handlers add a lot of complexity and I don't see a good use case for them. With the current approach user will be redirected to `primary-cta-link` anyways.

Can we stick with a simple approach? If someone asks for the onclick handler we probably will need to reconsider links implementation.

Reply:
Yeah makes sense. I have removed the handlers for now.